cinema/views.py

```diff
@@ -186,9 +264,37 @@
     authentication_classes = (TokenAuthentication,)
     permission_classes = (IsAuthenticated,)
 
+
+
     def get_queryset(self):
         return Order.objects.filter(user=self.request.user)
 
+
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "user",
+                type=OpenApiTypes.INT,
+                description="Id of the user."
+            )
+        ],
+        description="Or orders that belongs to the user withe the given id",
+        responses={
+            200: OpenApiResponse(OrderListSerializer, description="List of orders"),
+            201: OpenApiResponse(OrderSerializer),
+            400: "Bad request"
```

Comment:
The response description for status 400 should be wrapped in `OpenApiResponse` for consistency with other responses.